### PR TITLE
add ex6 about uses only the platformVersion

### PR DESCRIPTION
**Pull request by eyaly**
@wswebcreation - hope you can review it

### CODE REVIEW

File: .sauce/runner-ex6.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Example 5: Parallel execution using dynamic devices
```
Comment (wswebcreation) [inline .sauce/runner-ex6.yml]: Can you explain here what this config is doing?

File: .sauce/runner-ex6.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+sauce:
+  # This can be `eu-central-1` or  `us-west-1`
+  region: eu-central-1
+  concurrency: 10
+  metadata:
+    name: Testing XCUITEST Support
+    tags:
+      - ios
+      - xcuitest
+      - ex4
```
Comment (wswebcreation) [inline .sauce/runner-ex6.yml]: can you update this to `ex6`

File: .sauce/runner-ex6.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  build: "Sauce XCUITest Real Devices"
+
+xcuitest:
+  app: ./apps/SauceLabs.Mobile.Sample.XCUITest.App.ipa
+  testApp: ./apps/SwagLabsMobileAppUITests-Runner.app
+
+suites:
+  - name: "Sauce Labs XCUITest Tests from Example 5 - testSuccessfulLogin"
```
Comment (wswebcreation) [inline .sauce/runner-ex6.yml]: Can you explain what this suite is doing, also bump the example number

File: .sauce/runner-ex6.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    testOptions:
+      class: ["SwagLabsMobileAppUITests.LoginTests/testSuccessfulLogin"]
+    devices:
+      # Run on whatever iPhone device is available
+      - name: "iPhone.*"
+        platformVersion: "14.6"
+  - name: "Sauce Labs XCUITest Tests from Example 5 - testSuccessfulLogin"
```
Comment (wswebcreation) [inline .sauce/runner-ex6.yml]: Can you explain what this suite is doing, also bump the example number

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Hi @eyaly 

Can you fix the minor comments? Then we can merge it

**Comment by eyaly** [comment]
Thanks @wswebcreation for the comments. I fixed the code according to your comments.

**Review by wswebcreation — APPROVED** [review]
LGTM, thanks!

**Pull request merged**